src/inference/dev_api/cpp_interfaces/interface/ie_iplugin_internal.hpp

```diff
@@ -279,6 +280,31 @@ class INFERENCE_ENGINE_1_0_DEPRECATED INFERENCE_ENGINE_API_CLASS(IInferencePlugi
                                                                       const std::shared_ptr<RemoteContext>& context,
                                                                       const std::map<std::string, std::string>& config);
 
+    /**
+     * @brief Creates an executable network from an previously exported network using plugin implementation
+     *        and removes Inference Engine magic and plugin name
+     * @param networkBuffer Reference to network model buffer generated by mmap
+     * @param config A string -> string map of parameters
+     * @return An Executable network
+     */
+    virtual std::shared_ptr<IExecutableNetworkInternal> ImportNetwork(
+        std::shared_ptr<ov::util::MmapBuffer>& networkBuffer,
```

Comment:
I suppose we can create `istream` on top of MmapBuffer and avoid introducing new API?

Reply:
yes, we can do it to avoid adding additional new api.